CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Move all commonly used imports into application.scss (PR #683)
```

Comment:
Shouldn't it be _... into`_all_components.scss`_?

Reply:
Oh yeah, good point. Will fix.